Discussion/Discussion/Presentation/Comments/Responses/ResponsesViewModel.swift

```diff
@@ -13,9 +13,10 @@ import Combine
 public class ResponsesViewModel: BaseResponsesViewModel, ObservableObject {
     
     @Published var scrollTrigger: Bool = false
-    
     private let threadStateSubject: CurrentValueSubject<ThreadPostState?, Never>
-    
+
+    public var isBlackedOut: Bool = false
```

Comment:
Do we need to have it in viewModel? isn't it good to declare a variable in the view and use it directly

Reply:
I think better use only state variable in view, others variable in view model, and view model say how view should look